src/main/java/gov/nasa/pds/tools/validate/content/table/TableContentProblem.java

```diff
@@ -130,7 +154,7 @@ public Integer getTable() {
    * 
    * @return the record index.
    */
-  public Integer getRecord() {
+  public long getRecord() {
```

Comment:
What's the reason going from object `Integer` to primitive `long` and not object `Long`?

Reply:
@nutjob4life nothing in particular except I don't know the original rationale for it being `Integer` to begin with?

Reply:
I don't feel strongly either way.

Reply:
Auto-boxing and un-boxing for the win I guess 😁